src/components/Availability.js

```diff
@@ -37,6 +51,11 @@ export default function Availability({ entry }) {
                             }`}
                         </div>
                     ))}
+                    {totalAvailableSlots > 1 && availableSlots.length > 1 && (
```

Comment:
I updated this to `availableSlots.length > 1` because I noticed that if there's only 1 slot available, then the total is duplicative.
In my last commit, I had `> 0` so we get this:
![Screen Shot 2021-02-28 at 7 12 23 PM](https://user-images.githubusercontent.com/6521257/109438759-449eaf80-79f9-11eb-8208-03cb4e0b28a1.png)

And now with `> 1` we only see this case:
![Screen Shot 2021-02-28 at 7 12 15 PM](https://user-images.githubusercontent.com/6521257/109438778-57b17f80-79f9-11eb-9377-5e9f9906ae5d.png)